### PR TITLE
Add sentry capture exception

### PR DESCRIPTION
**Pull request by mariano-aguero**
Added captureException from sentry, in some critical points of the application, so we can log more error information in sentry

### CODE REVIEW

File: src/components/layouts/base/AppBar.vue [hunk 1]
```diff
@@ -154,6 +155,7 @@ export default {
         Vue.web3Provider = this.$web3Provider
       } catch (e) {
         this.$rLogin.clearCachedProvider()
+        Sentry.captureException(e)
```
Comment (pmprete) [inline src/components/layouts/base/AppBar.vue:158]: I think that if the user rejects it, you will get an error here, if that's the case there can be a lot of false positives, could you check if this is so?

File: src/main.js [hunk 2]
```diff
@@ -55,6 +55,7 @@ if (process.env.VUE_APP_SENTRY_DSN) {
     // We recommend adjusting this value in production, or using tracesSampler
     // for finer control
     tracesSampleRate: 1.0,
+    environment: process.env.VUE_APP_ENV,
```
Comment (pmprete) [inline src/main.js:58]: this enviroment variable does not exists, we could use NODE_ENV or create a new variable

**Review by pmprete — DISMISSED** [review]
Except my comments looks good, great work 👍

File: src/components/layouts/base/AppBar.vue (hunk 1 above)
Comment (mariano-aguero) [reply src/components/layouts/base/AppBar.vue]: If I close the  modal , rLogin triggers this exception..
Screenshot_20210221_150959

If I cancel the connection, rLogin will trigger this other oxception
Screenshot_20210221_151007

So, I think the best option is to remove the captureException method in this try..catch

File: src/main.js (hunk 2 above)
Comment (mariano-aguero) [reply src/main.js]: Fixed

**Review by pmprete — APPROVED** [review]
Good job!

**Pull request merged**